puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/pipeline/luna-nightly-rpm-pipeline.yaml

```diff
@@ -10,6 +10,7 @@
           result: success
     dsl:
       !include-raw:
+        - pipelines/vars/katello/nightly.groovy
```

Comment:
should luna have it's own vars?

Reply:
Doesn't need to but I can go that route

Reply:
ok, let's start with this and split it only if needed